core/src/bin/retryExports.ts

```diff
@@ -0,0 +1,169 @@
+import Moment from "moment";
+import { CLI, ParamsFrom } from "actionhero";
+import { GrouparooCLI } from "../modules/cli";
+import { APIData } from "../modules/apiData";
+import { Export } from "../models/Export";
+import { Destination } from "../models/Destination";
+
+export class RetryExportsCLI extends CLI {
+  name = "retryExports";
+  description =
+    "Marks failed Exports that were created in a given time frame to be retried.";
+  inputs = {
+    start: {
+      required: false,
+      requiredValue: true,
+      formatter: APIData.ensureDate,
+      description:
+        "Search for failed Exports created on or after this timestamp.",
+    },
+    startAgo: {
+      required: false,
+      requiredValue: true,
+      formatter: APIData.ensureNumber,
+      description:
+        "Search for failed Exports created on or after a certain number of seconds ago. See --startAgoUnit to choose a different time unit.",
+    },
+    startAgoUnit: {
+      required: false,
+      requiredValue: true,
+      default: "seconds",
+      description:
+        "Unit for --startAgo (e.g. seconds, minutes, hours, days...)",
+      formatter: (val: string) => {
+        const normalized = Moment.normalizeUnits(val as Moment.unitOfTime.Base);
+        return normalized as Moment.unitOfTime.Base;
+      },
+      validator: (val: string) => {
+        if (!val) throw new Error(`invalid unit of time`);
+      },
+    },
+    end: {
+      required: false,
+      requiredValue: true,
+      formatter: APIData.ensureDate,
+      description:
+        "Search for failed Exports created on or before this timestamp. Defaults to the current time.",
+    },
+    preview: {
+      default: false,
+      formatter: APIData.ensureBoolean,
+      description:
+        "When set, will not make any changes and only outputs the number of Exports that would be retried.",
+      flag: true,
+      letter: "p",
+    },
+    destinationIds: {
+      required: false,
+      requiredValue: true,
+      formatter: (val: any) => val as string[],
```

Comment:
We can remove this once actionhero is updated (swap out for a `variadic: true as const`)